templates/request/legal-info.tpl

```diff
@@ -0,0 +1,7 @@
+<div class="row">
+    <p class="legal-info">
+        By submitting this form, you agree to your information being processed to fulfil your request in
```

Comment:
Genuine question - does this legally matter? I can't recall if we have a defined preferred ENGVAR in the tool at all.

I can't imagine it being important immediately, but it might be something to come back to.

Reply:
Hmm, I'm really not sure. Technically the legal venue here would be the US, but I have a hard time believing that the use of British English would nullify the agreement.

I would say let's kick this down the road to a larger discussion of what ENGVAR we want to use; I can't see this being any issue in the near-term future. (Or if it is, I'm sure we'll find out about it.)